packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -220,6 +221,11 @@ export const esmPatientRegistrationSchema = {
         _default: true,
         _description: 'Whether to display capture patient photo slot on name field',
       },
+      displayFamilyNameFieldFirst: {
```

Comment:
```suggestion
      displayReverseFieldOrder: {
```

Reckon something like `displayReverseFieldOrder` or `reverseFieldOrder` might be a better fit here?

Reply:
I agree